dialects/postgresql/postgresql-test-fixtures/src/main/kotlin/app/cash/sqldelight/dialects/postgresql/PostgresqlTestFixtures.kt

```diff
@@ -0,0 +1,11 @@
+package app.cash.sqldelight.dialects.postgresql
+
+import java.io.File
+
+object PostgresqlTestFixtures {
+  val fixtures by lazy {
+    File("build/fixtures").listFiles()
```

Comment:
Hmmm this doesn't seem correct. This refers to the ansi fixtures.

If I change this to fixtures_postgresql it's empty.